### DISCUSSION TITLE
Organizational methodology?

### DISCUSSION THREAD

**Discussion by kubu4** [sr320/ceabigr#72] [Q&A] [answered]
@sr320 - Just curious about what the "method behind the madness" is in regards to repo organization. Specifically, how do you decide on subdirectory numbering? 

E.g.

`ceabigr/output/` has directories starting with numbers, but they're not sequential. Why is this?

I figured I'd add a new subdirectory, but wanted to keep the naming conventions consistent.

**Comment by sr320** [comment, marked as answer]
It is just about numbers - they are sequential in the sense I move up in numbers but do skip. 
The rationale is so we can better communicate and all output can be linked back to code directly. 
Rmd number corresponds with output number.

For instance - I can say it is in Rmd #42  
as opposed to some string of words that is often over used.

also I am old, and it is easy to remember numbers :)

**Reply by kubu4** [reply]
Ha! But why skip numbers????!!!! The number skipping is making me realize I have a touch of OCD...

**Reply by sr320** [reply]
Good Question - lets say I am working on differential methylation and on 37-- and want to switch focus completely to TE activity. I would jump up to 40 to give room for more methylation so similar efforts are together.
